.azdo/pipelines/app-ci.yaml

```diff
@@ -82,7 +84,7 @@ stages:
             inputs:
               azureSubscription: '<azureSubscriptionServiceConnection>'
               appType: 'webAppLinux'
-              appName: '<appServiceName>
+              appName: '$(azureAppServiceName)'
```

Comment:
have you tested "azureAppServiceName" is resolved to an external variable ? Has this variable been defined in a variable group like this: https://learn.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=classic